doc/source/Resources/pyaedt_installer_from_aedt.py

```diff
@@ -237,7 +234,7 @@ def install_pyaedt():
 
             run_command('"{}" install --no-cache-dir --no-index --find-links={} pyaedt'.format(pip_exe, unzipped_path))
```

Comment:
It should be here: 

run_command('"{}" install --no-cache-dir --no-index --find-links={} pyaedt[installer]'.format(pip_exe, unzipped_path))